eclass/ros-catkin.eclass

```diff
@@ -192,8 +195,15 @@ ros-catkin_src_configure_internal() {
 # @DESCRIPTION:
 # Configures a catkin-based package.
 ros-catkin_src_configure() {
-	export CATKIN_PREFIX_PATH="${EPREFIX}/usr"
-	export ROS_ROOT="${EPREFIX}/usr/share/ros"
+	if [ -f "${ROSPREFIX%/}/setup.bash" ]; then
```

Comment:
@mgorny what do you think about sourcing this ? shall it use a subshell or something ?